frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -88,22 +103,36 @@ export class MineTailingsInfo extends Component {
     updatedDocument.due_date = value.tsf_report_due_date;
     updatedDocument.received_date = value.tsf_report_received_date;
     updatedDocument.exp_document_status_guid = value.tsf_report_status;
-    this.props.updateExpectedDocument(updatedDocument.exp_document_guid, {document: updatedDocument}).then(() => {
-      this.props.closeModal();
-      this.props.fetchMineRecordById(this.props.mine.guid);
-    })
+    this.props
+      .updateExpectedDocument(updatedDocument.exp_document_guid, { document: updatedDocument })
+      .then(() => {
+        this.props.closeModal();
+        this.props.fetchMineRecordById(this.props.mine.guid);
+      });
   };
 
   openEditReportModal(event, onSubmit, title, statusOptions, doc) {
     this.setState({
-      selectedDocument: doc
-    })
+      selectedDocument: doc,
+    });
     event.preventDefault();
     const initialValues = {
-      "tsf_report_name": doc ? ((doc.exp_document_name === 'None') ? null : doc.exp_document_name) : null,
```

Comment:
Should avoid nested ternary, 
Since they are all checking the existence of `doc`, pull that out and create the initialValues if doc exists

Reply:
Agree this is really awkward. Not sure why it was done this way, but changing.

Reply:
Changed.